Tuist/ProjectDescriptionHelpers/Module.swift

```diff
@@ -388,90 +381,82 @@ public enum Module: String, CaseIterable {
                 .target(name: Module.automation.testingTargetName!),
                 .target(name: Module.migration.testingTargetName!),
                 .target(name: Module.asyncQueue.testingTargetName!),
-                .target(name: Module.xcodeProjectGenerator.testingTargetName!),
                 .target(name: Module.plugin.targetName),
                 .target(name: Module.plugin.testingTargetName!),
                 .external(name: "ArgumentParser"),
                 .external(name: "GraphViz"),
                 .external(name: "AnyCodable"),
-            ]
-        case .xcodeProjectGenerator:
-            [
-                .target(name: Module.core.targetName),
-                .target(name: Module.core.testingTargetName!),
-                .target(name: Module.support.targetName),
-                .target(name: Module.support.testingTargetName!),
-                .external(name: "XcodeProj"),
+                .external(name: "XcodeGraphTesting"),
```

Comment:
What do you think that as a part of this effort, we get rid of `XcodeGraphTesting` library in favor of putting test data behind the `#MOCKING` compiler flag? Definitely fine to do in a follow-up, though

Reply:
Yeah, we should. I'll do it in a follow-up PR.